index.js

```diff
@@ -32,7 +32,10 @@ const stickers = [
     'CAADBQAD-wADDxXNGe8aqxEu9OCLAg',
     'CAADBQAD_AADDxXNGSb44I6FN-UzAg',
     'CAADBQAD_QADDxXNGYGYV17DXBbkAg',
-    'CAADBQAD_gADDxXNGWuj_Z6psGN4Ag'
+    'CAADBQAD_gADDxXNGWuj_Z6psGN4Ag',
+    'CAADBQADCAAD7Dx2Ikbxv7doe-meAg', // still working!
+    'CAADBQADDAAD7Dx2IpKz4vfpPlrCAg', // stupid human!
+    'CAADBQADBQAD7Dx2IlqNSpG32Rg4Ag'  // get up!
```

Comment:
I don't think it make much sense to send `still working` and `stupid human` stickers.

Reply:
Feel free to adjust it by your idea, I used another branch to request a pull.

Reply:
This part is just for fun...That's why I decided to make another branch and keep this one more 'official'